clients/admin-ui/cypress/e2e/datasets.cy.ts

```diff
@@ -6,11 +6,8 @@ import {
 } from "cypress/support/stubs";
 
 describe("Dataset", () => {
-  before(() => {
```

Comment:
Note that `before` runs once for the describe block, instead of wrapping each test case. That's generally more fragile, and it was in this case because Cypress automatically clears localStorage between test cases.